stable/jenkins/templates/rbac.yaml

```diff
@@ -1,7 +1,7 @@
 {{ if .Values.rbac.install }}
 {{- $serviceName := include "jenkins.fullname" . -}}
 apiVersion: rbac.authorization.k8s.io/{{ required "A valid .Values.rbac.apiVersion entry required!" .Values.rbac.apiVersion }}
-kind: ClusterRoleBinding
+kind: {{ .Values.rbac.roleBindingKind }}
```

Comment:
If this is changed to rolebinding does the namespace key in the subjects array still work? Have you tested this?

Reply:
It does work, as far as I saw/tested. Are there some kind of automated tests I could extend?

Reply:
I did a few additional rounds of testing and it works without any issues.

Reply:
`roleRef.kind` should be `Role` instead of `ClusterRole`

Reply:
I guess that there should be an option to use either of the two kinds. I can work on it as well but I'd keep that as a separate PR.